config/initializers/ext_action_mailer.rb

```diff
@@ -2,7 +2,9 @@
 # can route between #deliver_now and #deliver_later
 module ActionMailer
   class MessageDelivery
+    alias_method :original_deliver_later, :deliver_later
```

Comment:
I just remembered we can use `prepend` to avoid `alias_method` dances in https://github.com/18F/identity-idp/pull/7165

So here we could define somewhere the file:

```ruby
module DeliverLaterArgumentChecker
  def deliver_later(...)
    MailerSensitiveInformationChecker.check_for_sensitive_pii!(@params, @args, @action)
    super(...)
  end
end
```

and turn this into:

```suggestion
    prepend DeliverLaterArgumentChecker
```

Reply:
Thanks for reminding me, added in https://github.com/18F/identity-idp/pull/7155/commits/b8f658bcb518a8b23a5701a1f5c39f76e29b4863